Sound/Tidal/Stream.hs

```diff
@@ -218,6 +218,11 @@ infixl 1 |/|
 (|/|) :: ParamPattern -> ParamPattern -> ParamPattern
 (|/|) = mergeNumWith (div) (/)
 
+(###) = foldl (#)
```

Comment:
although it might seem obvious for a haskell programmer what these do,
can you add example usages or short explanations as haddock comments for one of these at least?

still hoping we can somehow use these docs somewhere else as well....